### ISSUE TITLE
Sign Bank - searching by handshape doesn't work

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/ASL-DAI#11]
If no text string is included, the handshape search fails.  It should be possible to search for start and end handshapes without specifying text.

![screen shot 2017-10-04 at 12 29 29 pm](https://user-images.githubusercontent.com/13629122/31187434-ea4c0924-a8ff-11e7-8fc4-f7097e130dfd.png)

The problem seems to be, in part, that none of the radio boxes for 1-handed, 2-handed, or Either was selected. Either should be selected by default.   But with "either" selected plus a handshape, the search does not seem to restrict the results to the handshape selected.

![screen shot 2017-10-04 at 12 46 44 pm](https://user-images.githubusercontent.com/13629122/31188193-206d58ee-a902-11e7-9829-af88a27dd397.png)
![screen shot 2017-10-04 at 12 46 27 pm](https://user-images.githubusercontent.com/13629122/31188192-203fe418-a902-11e7-943f-f1b2c103b623.png)

Furthermore, when a lexical item is selected, and "include handshapes" is selected, the search is not restricted to the selected handshape.

![screen shot 2017-10-04 at 12 49 06 pm](https://user-images.githubusercontent.com/13629122/31188295-705e4ed0-a902-11e7-8a49-040f8a2152ed.png)
![screen shot 2017-10-04 at 12 49 01 pm](https://user-images.githubusercontent.com/13629122/31188296-7067039a-a902-11e7-8f96-2f6da3437dc4.png)

Basically it seems the the handshape search is just not working at all.

**Comment by kwasiopoku** [comment]
The handshape labels in the ASLLVD dataset is in one form while the continues sign data is in a different form.  I need to translate the ASLLVD handshape labels.  Should be pretty straight forward.  @cneidle  does this make sense to you and do you have the mapping handy?

**ASSLVD**
+------------------+
| D_START_HS_LABEL |
+------------------+
|                  |
| 1                |
| 10               |
| 25               |
| 3                |
| 4                |
| 5                |
| 5-C              |
| 5-C-L            |
| 5-C-tt           |
| 6                |
| 7                |
| 8                |
| A                |
| alt-G            |
| alt-M            |
| alt-N            |
| alt-P            |
| B                |
| B-L              |
| B-xd             |
| baby-O           |
| bent-1           |
| bent-B           |
| bent-B-L         |
| bent-B-xd        |
| bent-Horns       |
| bent-I-L-Y       |
| bent-M           |
| bent-N           |
| bent-U           |
| bent-U-L         |
| C                |
| cocked-7         |
| cocked-8         |
| cocked-F         |
| cocked-S         |
| cocked-U         |
| crvd-3           |
| crvd-5           |
| crvd-B           |
| crvd-flat-B      |
| crvd-L           |
| crvd-sprd-B      |
| crvd-U           |
| crvd-V           |
| crvd-W           |
| D                |
| E                |
| fanned-flat-O    |
| flat-B           |
| flat-G           |
| flat-O           |
| flat-O_2         |
| full-M           |
| F_9              |
| G_Q              |
| Horns            |
| I                |
| I-L-Y            |
| L                |
| L-X              |
| loose-E          |
| M                |
| N                |
| O                |
| open-7           |
| open-8           |
| open-F           |
| other            |
| O_2-Horns        |
| P_K              |
| R                |
| R-L              |
| S                |
| sml-C_3          |
| T                |
| tight-C          |
| tight-C_2        |
| U-L              |
| U_H              |
| V_2              |
| W                |
| X                |
| X-over-thumb     |
| Y                |
+------------------+

**Continuous Sign**

+------------------+
| D_START_HS_LABEL |
+------------------+
|                  |
| HSH_001          |
| HSH_002          |
| HSH_003          |
| HSH_004          |
| HSH_005          |
| HSH_006          |
| HSH_007          |
| HSH_010          |
| HSH_012          |
| HSH_013          |
| HSH_014          |
| HSH_017          |
| HSH_018          |
| HSH_019          |
| HSH_020          |
| HSH_021          |
| HSH_022          |
| HSH_023          |
| HSH_024          |
| HSH_026          |
| HSH_029          |
| HSH_030          |
| HSH_032          |
| HSH_035          |
| HSH_036          |
| HSH_038          |
| HSH_039          |
| HSH_040          |
| HSH_041          |
| HSH_042          |
| HSH_044          |
| HSH_045          |
| HSH_050          |
| HSH_051          |
| HSH_052          |
| HSH_053          |
| HSH_056          |
| HSH_057          |
| HSH_058          |
| HSH_059          |
| HSH_060          |
| HSH_061          |
| HSH_065          |
| HSH_066          |
| HSH_067          |
| HSH_071          |
| HSH_073          |
| HSH_074          |
| HSH_075          |
| HSH_076          |
| HSH_077          |
| HSH_078          |
| HSH_079          |
| HSH_080          |
| HSH_081          |
| HSH_082          |
| HSH_086          |
| HSH_091          |
| HSH_093          |
| HSH_094          |
| HSH_095          |
| HSH_097          |
+------------------+

**Comment by cneidle** [comment]
yes

http://www.bu.edu/asllrp/cslgr/pages/ncslgr-handshapes.html

**Comment by cneidle** [comment]
PS  Greg must also have a set of correspondences, since the images
in the handshapes folder distributed with SignStream have them labelled
very similarly to the ASLLVD labels (ignoring the “big-“ prefix):

**Comment by kwasiopoku** [comment]
Add fix for missing handshapes on view occurrences page.  
Hitting submit on the sign bank search form should return search results

**Comment by kwasiopoku** [comment]
All the issues on this ticket are fixed with the exception of the option to hit enter for submit.  We should probably close this ticket and open one just for the outstanding issue.

**Comment by kwasiopoku** [comment]
Actually the outstanding issue is being tracked with ticket #9

**Comment by cneidle** [comment]
As just mentioned on a related ticket, please verify that the dominant and non-dominant hand selectors are working properly in relation to handshape selection.   Thanks !!

**Comment by kwasiopoku** [comment]
Fixed
